lib/level/groups.ex

```diff
@@ -194,4 +210,19 @@ defmodule Level.Groups do
     |> Ecto.Changeset.change(state: "CLOSED")
     |> Repo.update()
   end
+
+  @doc false
```

Comment:
One thing you might consider here is using the `@impl true` annotation (and in other places where you're implementing this behaviour). That's when you're specifying which functions are implementations of behaviour callbacks. You can read a bit more about `@impl` in the 1.5 release notes: https://elixir-lang.org/blog/2017/07/25/elixir-v1-5-0-released/ and in the issue around its addition to the language: https://github.com/elixir-lang/elixir/issues/5734

Reply:
🤩 this is great!